src/Create.js

```diff
@@ -30,21 +31,36 @@ const useStyles = makeStyles((theme) => ({
 
 function Create(props) {
   const [grid, setGrid] = useState([]);
+  const [solution, setSolution] = useState(null);
+  const [user, setUser] = useState(null);
+  const defaultUserId = "kricheso@google.com";
   const classes = useStyles();
 
+  async function getCurrentUser() {
+    // User login only works after doing some other firebase call:
+    const _ = await Firestore.get.doesGridExist("null_id");
```

Comment:
Interesting... I wonder if this import line could help?

import firebaseConfig from './firebase'; // Must be present to initialize database.

Reply:
That's a good idea.  It doesn't fix it though.  I'll open an issue for it.

Reply:
Oh wait it looks like it could be a timing issue.  Apparently it has to be called after something is mounted?  Using setTimeout for a couple seconds is another work around.